From a9755b7ca244a7ac515e1f81603c8b4286178828 Mon Sep 17 00:00:00 2001
From: Vitaly Zaitsev <vitaly@easycoding.org>
Date: Sat, 23 Jul 2016 18:54:44 +0500
Subject: [PATCH 1/2] Install 'cutegram' binary when BinaryMode is enabled

Closes: #231
---
 cutegram.pro | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cutegram.pro b/cutegram.pro
index 2bb3824..d87ed00 100644
--- a/cutegram.pro
+++ b/cutegram.pro
@@ -2,6 +2,8 @@ contains(CONFIG, binaryMode) {
     CONFIG += c++11
     TARGET = cutegram
     QT += qml quick gui widgets core
+    target.path = $$PREFIX/bin
+    INSTALLS += target
     TEMPLATE = app
     SOURCES += main.cpp
     RESOURCES += \
-- 
2.9.2

