From 1c2f4449084811e66dfa2d1a73c2849e08ecfd9f Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <i.gnatenko.brain@gmail.com>
Date: Sat, 23 Jul 2016 20:02:21 +0200
Subject: [PATCH 3/3] don't install qmlFiles when binaryMode is enabled

Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>
---
 cutegram.pro | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/cutegram.pro b/cutegram.pro
index 9bbe2dd..fffea20 100644
--- a/cutegram.pro
+++ b/cutegram.pro
@@ -2,8 +2,6 @@ contains(CONFIG, binaryMode) {
     CONFIG += c++11
     TARGET = cutegram
     QT += qml quick gui widgets core
-    target.path = $$PREFIX/bin
-    INSTALLS += target
     TEMPLATE = app
     SOURCES += main.cpp
     RESOURCES += \
@@ -49,4 +47,10 @@ qmlFiles.files = \
     LICENSE
 
 QMAKE_SUBSTITUTES += shortcut
-INSTALLS += qmlFiles shortcut icons pixmaps
+contains(CONFIG, binaryMode) {
+    target.path = $$PREFIX/bin
+    INSTALLS += target
+} else {
+    INSTALLS += qmlFiles
+}
+INSTALLS += shortcut icons pixmaps
-- 
2.9.2

